### DISCUSSION TITLE
Possible problem with Vercel CLI 30.1.1

### DISCUSSION THREAD

**Discussion by jeffrafter** [vercel/community#2713] [Bug Report]
### Page to Investigate

/

### Steps to Reproduce

Our deploy relies on the latest version of Vercel CLI and just started breaking. I suspect something changed between Vercel CLI 30.1.0 (which works) and Vercel CLI 30.1.1 (which doesn't). We're seeing:

```
> Downloading project settings
Downloaded project settings to .vercel/project.json [0ms]
Vercel CLI 30.1.1
Error: (0 , build_utils_1.validateNpmrc) is not a function
Error: Process completed with exit code 1.
```

cc maybe @ijjk ? <3

**Comment by jeffrafter** [comment]
For anyone following along if you `npm install --global vercel@30.1.0` this will likely get you around the issue.

**Comment by amyegan** [comment]
Thanks for reporting this! Version `30.1.2` was released to address this issue with `30.1.1`, so you no longer need to stay on the older version `30.1.0`

**Reply by jeffrafter** [reply]
You are awesome thank you 🙌
